openslides/motions/static/js/motions/site.js

```diff
@@ -1054,7 +1054,7 @@ angular.module('OpenSlidesApp.motions.site', ['OpenSlidesApp.motions', 'OpenSlid
         Motion.loadRelations(motion, 'agenda_item');
         $scope.version = motion.active_version;
         $scope.isCollapsed = true;
-        $scope.commentsFields = MotionComment.getFields();
+        $scope.commentsFields = $scope.config('motions_comments');
```

Comment:
If you use Config.get(..) `MotionComment` provider is unused in MotionDetailCtrl and can be removed. @normanjaeckel Is it ok for you to use Config.get(..) here?